docs/src/content/ignition/docs/guides/creating-modules.md

```diff
@@ -169,6 +169,8 @@ The third and fourth parameters are both optional. They represent the amount of
 
 ### Deploying a library
 
+[ definition of "library".... I literally have no idea what this means. Library is an extremely abstract word in english ]
```

Comment:
If you want to take a stab, this is what we mean: https://docs.soliditylang.org/en/latest/contracts.html#libraries